docs/REST API/rest-api.md

```diff
@@ -3663,6 +3665,114 @@ OS vulnerability data is currently available for Windows and macOS. For other pl
 
 ```
 
+### Get host's software
+
+`GET /api/v1/fleet/hosts/:id/software/titles`
```

Comment:
```suggestion
`GET /api/v1/fleet/hosts/:id/software`
```

Can we simplify this? Even if we add versions later we can use `/versions`

Reply:
Makes sense to me.